### DISCUSSION TITLE
Expose everything to public or not?

### DISCUSSION THREAD

**Discussion by Shinigami92** [poolifier/poolifier#123] [Ideas]
https://github.com/pioardi/poolifier/pull/116#discussion_r574497757

**Comment by jerome-benoit** [comment]
I think we will try do avoid it as much as possible. The strategy selection should only used public attributes to give the worker to choose index in the pool worker array. That's doable but it requires a small rewrite.

**Reply by jerome-benoit** [reply]
I do not remember the initial scope of the abstract pool class properties. 
Most of them were public, right?

**Reply by Shinigami92** [reply]
We use git... you know... you could look it up in the history :smile:

**Reply by pioardi** [reply]
You can easily check that looking for our tags, we have one tag per release so you can look at the 1.2.1 version code in few seconds ;)
